tests/functional/base_test.go

```diff
@@ -349,6 +349,15 @@ func GetDefaultOpenStackControlPlaneSpec() map[string]interface{} {
 	ironicTemplate := map[string]interface{}{
 		"ironicConductors": []interface{}{},
 	}
+	cinderTemplate := map[string]interface{}{
+		"apiTimeout": 60,
+	}
+	glanceTemplate := map[string]interface{}{
+		"apiTimeout": 60,
+	}
+	manilaTemplate := map[string]interface{}{
+		"apiTimeout": 60,
+	}
```

Comment:
Should we remove these and rely on Cinder, Glance and Manila webhooks instead?  i.e. `go get -u` those updated APIs in this PR?  Or do we already have them perhaps?

Reply:
We can just deal with this later, regardless.  Better to just keep things moving forward to unblock CI.